dev-server/ui-playground/PlaygroundApp.js

```diff
@@ -83,23 +43,25 @@ export default function PlaygroundApp() {
   return (
     <main className="PlaygroundApp">
       <div className="PlaygroundApp__sidebar">
-        <a
-          className="PlaygroundApp__nav-link u-center"
-          href={baseUrl}
-          onClick={e => navigate(e, '/')}
-        >
-          <SvgIcon name="logo" />
-        </a>
-        {demoRoutes.map(c => (
-          <a
-            className="PlaygroundApp__nav-link"
-            key={c.route}
-            href={c.route}
-            onClick={e => navigate(e, c.route)}
-          >
-            {c.title}
+        <div className="PlaygroundApp__sidebar-home">
+          <a href={baseUrl} onClick={e => navigate(e, '/')}>
+            <SvgIcon name="logo" />
           </a>
-        ))}
+        </div>
+        <ul>
+          {demoRoutes.map(c => (
+            <li key={c.route}>
+              <a
```

Comment:
Minor issue, but I am not seeing the focus working at all on this element. Even tinkering around, I can't get it working. It seems like its not honoring not :focus-visible for some reason. In other words, tabbing to this element always actives the  `:focus:not(:focus-visible)` rule which makes no sense because it should be focus-visible :\